src/registrar/templates/includes/header_extended.html

```diff
@@ -91,12 +91,12 @@
             </li>
             {% endif %}
 
-            {% if has_organization_members_flag and has_view_members_portfolio_permission %}
```

Comment:
Blocking? -- Actually, we need to also check for the view/edit members permissions because these are not inherent to admin users.  Even if the members flag is on, the user might not have the right permissions to view members.  I think this was specified in Issue #2725

Reply:
yes